examples/meta/generator/targets/cpp.json

```diff
@@ -12,6 +12,7 @@
     "Init": {
         "Construct": "auto $name = some<C$typeName>($arguments)$kwargs",
         "Copy": "auto $name = wrap($expr)$kwargs",
+        "CopyRaw": "auto $name = $expr$kwargs",
```

Comment:
probably not the best name as nothing is being copied

Reply:
change it? :)